docs/web/docs/1-getting_started/3-aws.mdx

```diff
@@ -499,17 +496,19 @@ account references to match your environment:
 ```yaml
 template_type: NOQ::AWS::IdentityCenter::PermissionSet
 identifier: iambic_test_permission_set
-included_orgs:
-  - o-12345678
+# If we have multiple orgs, we can optionally limit the AWS Organization
+# to deploy a permission set to
+# included_orgs:
+#  - o-12345678
 access_rules:
```

Comment:
We may have to comment out the keyword access_rules: as well. We are very NoneType friendly in validators:

pydantic.error_wrappers.ValidationError: 1 validation error for AwsIdentityCenterPermissionSetTemplate
AccessRules
  'NoneType' object is not iterable (type=type_error)

I wonder how we can systematically validate all templates we use on docs.